code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -111,7 +111,7 @@ var/list/ai_list = list()
 
 	aiMulti = new(src)
 	aicamera = new/obj/item/device/camera/ai_camera(src)
-
+	eyeobj.addHear()
```

Comment:
No, as long as New() calls ..() up to atom/movable/New(), having flags = HEAR is enough to make sure it gets a virtualhearer